### PR TITLE
Use yaml.safe_load

### PR DESCRIPTION
**Pull request by benbalter**
/cc @ptoomey3

### CODE REVIEW

File: lib/licensee/license.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def content
 
     # License metadata from YAML front matter
     def meta
-      @meta ||= YAML.load(parts[1]) if parts && parts[1]
+      @meta ||= YAML.safe_load(parts[1]) if parts && parts[1]
```
Comment (ptoomey3) [inline lib/licensee/license.rb]: Just because you're paranoid doesn't mean they aren't after you.

**Comment by mislav** [comment]
Does this drop support for older Ruby versions? Do we care?

**Comment by benbalter** [comment]
Believe it's Ruby >= 2.1. I'm fine with that, but I'll defer to y'all, as y'all are much better Rubyists than I. 

Alternatively we could check if the method exists and fall back to `Yaml.load`.

**Comment by mislav** [comment]
Good point re: major bump.

I feel that it's worth checking whether the method exists. It's a bummer that this detail prevents Ruby 2.0 from using this library.

**Comment by ptoomey3** [comment]
> I feel that it's worth checking whether the method exists. It's a bummer that this detail prevents Ruby 2.0 (preinstalled on OS X) from using this library.

This seems super reasonable given that the safe yaml use is purely a defensive paranoia thing to prevent any misuse/bugs in the future. As written now, there appears to be no risk.

**Pull request merged**